### ISSUE TITLE
Price sort by logic 

### ISSUE THREAD

**Issue by exezbcz** [kodadot/nft-gallery#8890]
so, currently, our price filter, either low to high or high to low only work when buy now filter is applied 

i don't know what tunel vision I had, but it should work differently

![image](https://github.com/kodadot/nft-gallery/assets/90852205/953f5674-ea38-4a8f-a627-c52dff5eab9a)

## changes
- since its "sort by" it should actually sort without any other filter applied 
- when the low to high filter is active, show listed first, sorted by the sort by filter, and then leave the rest untouched. It takes all the supply and just reorders it so the listed ones are on top.
- then, when "buy now" is active as well, it filters out only the listed

**sort by price + buy now = listed items sorted by price** 

**sort by price = all items sorted based on price, non listed are there as well, but only after the listed ones**

**lets also rename "buy now" to listed** 

price range should work the same

![image](https://github.com/kodadot/nft-gallery/assets/90852205/ae0bfbeb-8825-4610-8014-d4b1357aa84d)
- it does not show the ones with 1 dot first

probably this needs to be changed, sorry for that 
- https://github.com/kodadot/nft-gallery/pull/8809

**Comment by hassnian** [comment]
👋

**Comment by kodabot** [comment]
ASSIGNED - @hassnian 🔒 LOCKED -> Wednesday, January 10th 2024, 21:46:45 UTC -> 36 hours

**Comment by yangwao** [comment]
we can reflect here to have  **sort buy now from low to high** *as default*

**Comment by yangwao** [comment]
> we can reflect here to have **sort buy now from low to high** _as default_

can it be done without doing https://github.com/kodadot/nft-gallery/pull/8920 ?

**Comment by exezbcz** [comment]
Possible to finish? 

cc @hassnian @vikiival

**Comment by vikiival** [comment]
> Possible to finish?

Hey we are waiting for a release of - https://github.com/subsquid/squid-sdk/pull/270

cc @eldargab can know more

**Comment by exezbcz** [comment]
cc @kodadot/internal-dev 


any takers here? not prio but would be nice to have till end of month

**Comment by vikiival** [comment]
Imo old @hassnian's PR can be reopened
